ts/store/reducers/entities/services/index.ts

```diff
@@ -225,4 +227,25 @@ export const notSelectedServicesSectionsSelector = createSelector(
   }
 );
 
+// Get the number of visible services that are not yet marked as read
+export const servicesBadgeValueSelector = createSelector(
+  [
+    nationalServicesSectionsSelector,
+    selectedLocalServicesSectionsSelector,
+    isFirstVisibleServiceLoadCompletedSelector
+  ],
+  (
+    nationalService,
+    localService,
+    isFirstVisibleServicesLoadCompleted
+  ) => {
+    let result = 0;
+    if (isFirstVisibleServicesLoadCompleted) {
+      nationalService.length > 0 && nationalService.forEach(service => result += service.data.length)
```

Comment:
Is the length check needed? You are using forEach.

Reply:
in this commit https://github.com/teamdigitale/io-app/commit/2131cbba7eb43b67c2428d4d92304cb1564e4641 I no longer use the forEach function